runtimes/runtimes/standalone.ts

```diff
@@ -166,20 +166,19 @@ export const standalone = (props: RuntimeProps) => {
                 }
             },
             fs: {
-                copy: (src, dest) => {
-                    const destDir = path.dirname(dest)
-                    if (!existsSync(destDir)) {
-                        mkdirSync(destDir, { recursive: true })
+                copyFile: (src, dest, options?) => {
```

Comment:
tbh, this also means we are not being consistent with node fs api. There is no options object in the copyFile function and now we are exposing an options object which implicitly has an `ensureDir` field. 

Imo it is better to either keep the copy function (which doesn't have a 1:1 mapping to node and allows us to be more flexible in the behaviour or the function) Or alternatively we expose copyFile as is and expect servers to do the exists, mkdir logic all on the server side. To me the second option looks worse at this point but might be the better option for the longer term

Reply:
I decided to go with `copyFile` name with `ensureDir` optional parameter set to false. In that case, default behavior of `copyFile` is aligned with Node.js, we provide additional useful feature for creating directory, and can be extended in the future to add optional `mode` parameter like in Node.js. Also, the behavior can be easily deduced from the method's name.